Makefile.am

```diff
@@ -250,8 +251,22 @@ systemdservicedir   = $(systemdsystemunitdir)
 systemdservice_DATA = $(systemdservice_files)
 endif
 
+if AUTOGOPATH
+export GOPATH := $(CURDIR)/.gopath
```

Comment:
Oh - just spotted that this is writing to cwd - this isn't going to be reliable - we should really use something like:

```
export GOPATH := $(mktemp -d --suffix=-cor-gopath)
```